Package.swift

```diff
@@ -2,6 +2,9 @@
 import PackageDescription
 let package = Package(
     name: "mongo-swift-driver",
+    platforms: [
+        .macOS(.v10_14)
```

Comment:
should this also list `v10_15`?

Reply:
confusingly, this list is meant to indicate minimum required platforms, not all supported platforms. having both of them gives an error about multiple declarations for the platform. see: https://github.com/apple/swift-package-manager/blob/master/Sources/PackageDescription/SupportedPlatforms.swift#L57

> /// Packages can choose to configure the minimum deployment
/// target version for a platform by using the APIs defined in this struct. The
/// Swift Package Manager emits appropriate errors when an invalid value is
/// provided for supported platforms, such as an empty array, multiple declarations
/// for the same platform, or an invalid version specification.

Reply:
the `SupportedPlatform` struct is also not to be confused with the `Platform` struct, which is used in other contexts in package manifests, e.g. how we use it to conditionally define `cSettings`: https://github.com/mongodb/mongo-swift-driver/blob/7e60c3c5a5fad0a6f1ca55170eed0909d2c7d062/Package.swift#L25

Reply:
Yeah, I think I had it confused with `Platform`.